lampod/src/chain/blockchain.rs

```diff
@@ -27,13 +37,59 @@ impl LampoChainManager {
         LampoChainManager {
             backend: client,
             wallet_manager,
+            current_height: Arc::new(Mutex::new(Height::from_consensus(0).unwrap())),
+            best_height: Arc::new(Mutex::new(Height::from_consensus(0).unwrap())),
```

Comment:
Please add a safety comment if call unwrap is safe and do not crash the program, otherwise return the error to the caller